jacquard/experiments/constraints.py

```diff
@@ -216,3 +216,23 @@ def matches_user(self, user, context=None):
             return False
 
         return True
+
+    def is_disjoint_from_constraints(self, other_constraints):
+        """
+        Test whether the set of users matching ``self``, and the set matching
+        ``constraints`` are disjoint.
```

Comment:
Would prefer a one-line subject and further details below—alas this is currently a linter error.

Reply:
1a253dd